woocommerce-google-analytics-integration.php

```diff
@@ -204,6 +204,84 @@ public function maybe_show_ga_pro_notices() {
 			WC_Admin_Notices::add_custom_notice( 'woocommerce_google_analytics_pro_notice', $notice_html );
 			update_option( 'woocommerce_google_analytics_pro_notice_shown', true );
 		}
+
+		/**
+		 * Get the path to something in the plugin dir.
+		 *
+		 * @param string $end End of the path.
+		 * @return string
+		 */
+		public function path( $end = '' ) {
+			return untrailingslashit( dirname( __FILE__ ) ) . $end;
+		}
+
+		/**
+		 * Get the URL to something in the plugin dir.
+		 *
+		 * @param string $end End of the URL.
+		 *
+		 * @return string
+		 */
+		public function url( $end = '' ) {
+			return untrailingslashit( plugin_dir_url( plugin_basename( __FILE__ ) ) ) . $end;
+		}
+
+		/**
+		 * Get the URL to something in the plugin admin assets dir.
+		 *
+		 * @param string $end End of the URL.
+		 *
+		 * @return string
+		 */
+		public function get_js_asset_url( $end = '' ) {
+			return $this->url( '/assets/js/build/' . $end );
+		}
+
+		/**
+		 * Get the path to something in the plugin admin dir.
+		 *
+		 * @param string $end End of the path.
+		 * @return string
+		 */
+		public function get_js_asset_path( $end = '' ) {
+			return $this->path( '/assets/js/build/' . $end );
+		}
+
+		/**
+		 * Gets the asset.php generated file for an asset name.
+		 *
+		 * @param string $asset_name The name of the asset to get the file from.
+		 * @return array The asset file. Or an empty array if the file doesn't exist.
+		 */
+		public function get_js_asset_file( $asset_name ) {
+			try {
+				return require $this->get_js_asset_path( $asset_name . '.asset.php' );
```

Comment:
Only external handles imported via the `import` declarations will be added to the `dependencies` in the *.asset.php file. This won't resolve the `$deps` for JS files that use `jQuery` directly.

Reply:
I added a workaround here https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/261/commits/450963ebb6e324df566f5c56c7bf8eab298ef00d